src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -960,6 +960,107 @@ default int getMaxEmotes()
+     */
+    @Nonnull
+    @CheckReturnValue
+    RestAction<List<GuildScheduledEvent>> retrieveScheduledEvents();
+
+    /**
+     * Creates a new {@link GuildScheduledEvent}.
+     * <p><b>Requirements</b></p>
+     * <p>Events that are created are required to have a name, a location, and a start time. Depending on the
+     * type of location provided, an event will be of one of three different {@link GuildScheduledEvent.Type Types}:
+     * <ol>
+     *     <li>
+     *         <p><b>{@link GuildScheduledEvent.Type#STAGE_INSTANCE Type.STAGE_INSTANCE}</b></p>
+     *         <p>These events are set to take place inside of a {@link net.dv8tion.jda.api.entities.StageChannel}. The
+     *         following permissions are required in the specified stage channel in order to create an event there:
+     *          <ul>
+     *              <li>{@link net.dv8tion.jda.api.Permission#MANAGE_EVENTS Permission.MANAGE_EVENTS}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL Permission.MANAGE_CHANNEL}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#VOICE_MUTE_OTHERS Permission.VOICE_MUTE_OTHERS}</li>
+     *              <li>{@link net.dv8tion.jda.api.Permission#VOICE_MOVE_OTHERS Permission.VOICE_MOVE_OTHERS}}</li>
+     *         </ul></p>
+     *     </li>
+     *     <li>
+     *         <p><b>{@link GuildScheduledEvent.Type#VOICE Type.VOICE}</b></p>
+     *         <p>These events are set to take place inside of a {@link net.dv8tion.jda.api.entities.VoiceChannel}. The
+     *         following permissions are required in the specified voice channel in order to create an event there:
+     *         <ul>
+     *             <li>{@link net.dv8tion.jda.api.Permission#MANAGE_EVENTS Permission.MANAGE_EVENTS}</li>
+     *             <li>{@link net.dv8tion.jda.api.Permission#VIEW_CHANNEL Permission.VIEW_CHANNEL}</li>
+     *             <li>{@link net.dv8tion.jda.api.Permission#VOICE_CONNECT Permission.VOICE_CONNECT}</li>
+     *         </ul></p>
+     *     </li>
+     *     <li>
+     *         <p><b>{@link GuildScheduledEvent.Type#EXTERNAL Type.EXTERNAL}</b></p>
+     *         <p>These events are set to take place at a custom location. {@link net.dv8tion.jda.api.Permission#MANAGE_EVENTS Permission.MANAGE_EVENTS}
+     *         is required on the guild level in order to create this type of event. Additionally, an end time <i>must</i>
+     *         also be specified.
+     *     </li>
+     * </ol>
+     * <p><b>Example</b></p>
+     * <pre>{@code
+     *     guild.createScheduledEvent()
+     *          .setName("Cactus Beauty Contest")
+     *          .setDescription("Come and have your cacti judged! _Must be spikey to enter_")
+     *          .setStartTime(OffsetDateTime.parse("2023-11-15T03:00:00+01:00"))
+     *          .setEndTime(OffsetDateTime.parse("2023-11-15T05:45:00+01:00"))
+     *          .setLocation("Mike's Backyard")
+     *          .queue();
```

Comment:
```suggestion
     * guild.createScheduledEvent()
     *     .setName("Cactus Beauty Contest")
     *     .setDescription("Come and have your cacti judged! _Must be spikey to enter_")
     *     .setStartTime(OffsetDateTime.parse("2023-11-15T03:00:00+01:00"))
     *     .setEndTime(OffsetDateTime.parse("2023-11-15T05:45:00+01:00"))
     *     .setLocation("Mike's Backyard")
     *     .queue();
```

I don't think that using `parse("...")` here is a good example. I would prefer using explicit constructors for `OffsetDateTime` with date and time fields instead.